src/firefighter/slack/models/user.py

```diff
@@ -451,12 +451,22 @@ def send_private_message(
         **kwargs: Any,
     ) -> None:
         """Send a private message to the user."""
-        client.chat_postMessage(
```

Comment:
We probably want to keep the behaviour of `send_private_message` to not save the message to DB: it is used to send errors to users for instance.

I'll let you decide between adding a new function, or adding an argument to this function :)